matter_server/server/device_controller.py

```diff
@@ -1076,15 +1090,15 @@ async def _setup_node(self, node_id: int) -> None:
             ):
                 try:
                     await self.interview_node(node_id)
-                except (NodeNotResolving, NodeInterviewFailed) as err:
+                except (NodeNotResolving, NodeInterviewFailed, TimeoutError) as err:
```

Comment:
Can you log Timeout errors separately here..

Reply:
it is being logged as the exception is sent to the logger

Reply:
Right :see_no_evil: . So this case is fine! :+1: